Client/Application/AppDelegate.swift

```diff
@@ -37,6 +37,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             // Use a clean profile for each test session.
             profile = BrowserProfile(localName: "testProfile", app: application)
             profile.files.removeFilesInDirectory()
+            profile.prefs.clearAll()
```

Comment:
Why must this be in `UIApplication`? Can't this live in a `XCTestCase` base class?